camus-etl-kafka/src/main/java/com/linkedin/camus/etl/kafka/common/KafkaReader.java

```diff
@@ -41,6 +39,8 @@
 public class KafkaReader {
 	// index of context
 	private static Logger log =  Logger.getLogger(KafkaReader.class);
+	private final int[] DELAYS_MS = {2, 512, 1024, 2048, 4096, 8192, 16384};
```

Comment:
Looks like a "magic intervals" for me. Maybe should we use usual "exponential backoff" algorithm like (2^tryCount) or any other variation with coefficients (if needed)?